packages/orchestration/src/exos/stakingAccountKit.js

```diff
@@ -189,52 +271,81 @@ export const prepareStakingAccountKit = (baggage, makeRecorderKit, zcf) => {
         /**
          * _Assumes users has already sent funds to their ICA, until #9193
          * @param {CosmosValidatorAddress} validator
-         * @param {Amount<'nat'>} ertpAmount
+         * @param {AmountArg} amount
          */
-        async delegate(validator, ertpAmount) {
-          trace('delegate', validator, ertpAmount);
-
-          // FIXME brand handling and amount scaling #9211
-          trace('TODO: handle brand', ertpAmount);
-          const amount = {
-            amount: String(ertpAmount.value),
-            denom: this.state.bondDenom,
-          };
-
-          const account = this.facets.helper.owned();
-          const delegatorAddress = this.state.chainAddress.address;
+        async delegate(validator, amount) {
+          trace('delegate', validator, amount);
+          const { helper } = this.facets;
+          const { chainAddress } = this.state;
 
-          const result = await E(account).executeEncodedTx([
+          // FIXME get values from proposal or args
+          const result = await E(helper.owned()).executeEncodedTx([
             toAnyJSON(
               MsgDelegate.toProtoMsg({
-                delegatorAddress,
+                delegatorAddress: chainAddress.address,
                 validatorAddress: validator.address,
-                amount,
+                amount: helper.amountToCoin(amount),
               }),
             ),
           ]);
 
-          if (!result) throw Fail`Failed to delegate.`;
-          return tryDecodeResponse(result, MsgDelegateResponse.fromProtoMsg);
+          const response = tryDecodeResponse(
+            result,
+            MsgDelegateResponse.fromProtoMsg,
+          );
+          mustMatch(harden(response), harden({})); // TODO: ignore response??
```

Comment:
if we're not returning anything, is decoding the response worthwhile?

Reply:
i don't think so. if it does start returning something this code can continue to ignore it